providers/certification-client/units/client-cert-odm-ubuntucore-18.pxu

```diff
@@ -46,6 +46,7 @@ nested_part:
     after-suspend-ethernet-manual
     after-suspend-wwan-manual
     after-suspend-thunderbolt-cert-manual 
+    acpi_oem_osi_strings
```

Comment:
If you implement a `acpi-manual` test plan, you can put it here instead of `acpi_oem_osi_strings` (
it's better to put it higher in the list, maybe above `audio-manual`):

```suggestion
    acpi-manual
```


If you don't, then you can just remove `acpi_oem_osi_strings`.